lib/middleware.js

```diff
@@ -33,10 +33,10 @@ var checkImports = function(path, next) {
   var changed = [];
 
   nodes.forEach(function(imported){
-    fs.stat(imported.path, function(err, stat) {
```

Comment:
I need to do some more testing, but I think I may have broke something the `imported.path` change, so I am going to hold off on this until I test it.